icu4c/source/i18n/casetrn.h

```diff
@@ -58,7 +58,7 @@ class CaseMapTransliterator : public Transliterator {
      * Transliterator API.
      * @return a copy of the object.
      */
-    virtual CaseMapTransliterator* clone() const = 0;
+    //virtual CaseMapTransliterator* clone() const = 0;
```

Comment:
Same question about this being commented out.

Reply:
virtual CaseMapTransliterator* clone() const override = 0;